tests/operators/test_python.py

```diff
@@ -999,6 +999,21 @@ def test_lambda(self):
         with pytest.raises(AirflowException):
             PythonVirtualenvOperator(python_callable=lambda x: 4, task_id='task', dag=self.dag)
 
+    def test_system_site_packages_with_wildcard_callable(self):
+        def mycallable(**kwargs):
```

Comment:
This is...... surprising. Like why would `**kwargs` and site packages cause this behaviour?!

Reply:
Spent time on it trying to figure out why but it seems that with site packages, the many airflow pickleable objects are no longer pickleable. If you remove everything from https://github.com/apache/airflow/blob/2d6282d6b7d8c7603e96f0f28ebe0180855687f3/airflow/operators/python.py#L358 except config and params, it works with site packages

Reply:
Ohhhhh, it's cos of what we pass in those kwargs. That makes sense.